index.js

```
@@ -14,7 +14,9 @@ module.exports = {
         'global-strict': 0,
         'indent': 2,
         'key-spacing': 0,
-        'max-len': [2, 120, 4],
+        'max-len': [2, 120, 4, {
+            "ignorePattern": "^(import\\s.+)|(\\s*(var\\s.+=\\s*)?require\\s*\\(.+)$"
```

I left it out to only match `require(`, which lets me get away with not needing decide on which quotes. Anything without a closing paren should throw a syntax error